packages/sdk/client/src/services/worker-client-services.ts

```diff
@@ -47,11 +49,21 @@ export class WorkerClientServices implements ClientServicesProvider {
   private _runtime!: SharedWorkerConnection;
   private _services!: ClientServicesProxy;
   private _loggingStream?: Stream<LogEntry>;
-
-  constructor({ config, createWorker, logFilter = 'error,warn' }: WorkerClientServicesParams) {
+  private readonly _observabilityGroup?: string;
+  private readonly _signalTelemetryEnabled: boolean;
+
+  constructor({
+    config,
+    createWorker,
+    logFilter = 'error,warn',
+    observabilityGroup,
+    signalTelemetryEnabled,
+  }: WorkerClientServicesParams) {
     this._config = config;
     this._createWorker = createWorker;
     this._logFilter = parseFilter(logFilter);
+    this._observabilityGroup = observabilityGroup;
+    this._signalTelemetryEnabled = signalTelemetryEnabled ?? false;
```

Comment:
having the default for this be false everywhere seems like it defeats the goal of introducing it?

that being said, if we want the default to be true, should probably consider how to make it clear to developers that its happening so that they can choose to opt out. cc @jessmartin @zhenyasav

Reply:
I intended to use the same observability opt-out decision from the app, but I forgot to reverse the condition.

So the default should be on unless the user opts out.